src/Raven.Server/Documents/Sharding/ShardedDocumentsStorage.cs

```diff
@@ -363,6 +367,9 @@ public ShardedDocumentDatabase.DeleteBucketCommand.DeleteBucketResult DeleteBuck
             // delete revisions for document
             RevisionsStorage.ForceDeleteAllRevisionsFor(context, document.Id);
             deleted++;
+
+            if (context.Transaction.InnerTransaction.LowLevelTransaction.TransactionSizeInPages > MaxTransactionSize)
```

Comment:
Here you are checking that the transaction size is great than 128GB, no?
That is likely not what you meant. `MaxTransactionSize` is in bytes, `TransactionSizeInPages` is in pages